Aztec/Classes/TextKit/ParagraphProperty/TextList.swift

```diff
@@ -14,10 +14,21 @@ open class TextList: ParagraphProperty {
         case ordered
         case unordered
 
-        func markerText(forItemNumber number: Int) -> String {
+        func markerText(forItemNumber number: Int, level: Int) -> String {
             switch self {
             case .ordered:      return "\t\(number)."
-            case .unordered:    return "\t\u{2022}"
+            case .unordered:    return unorderedMarker(for: level)
```

Comment:
Is the code here related to the bug fix?  I'm not sure I follow how the logic here affects the issue at hand.

Reply:
**TL:DR:** Not at all.

**Long story:**
I added the level information to `TextList` while searching of a way to fix the original bug. When I noticed that it wasn't necessary, instead of removing it, I saw the opportunity to implement the bullets-per-level support. On gutenberg they use a different bullet for indented unordered lists (`{25E6}`), so I thought it would be a good addition.

Talking with @hypest about it, we decided to keep the old one until Aztec-Android also has support for different bullet per indentation level.

We could move this code to a different PR if it's necessary. No problem with that! 👍

Reply:
I hate to be adding extra work to your plate, but I'd encourage moving the remaining changes to a different PR indeed, for documentation and history purposes - and also to provide more clarity to this PR for future reference.